sketch_css/src/sketch/css.gleam

```diff
@@ -463,17 +465,24 @@ fn property_body(param) {
     [Field(None, p)] -> {
       case rewrite_pipe(p) {
         Call(FieldAccess(Variable(_), size), [Field(None, value)]) ->
-          size_to_string(value) <> size
+          size_value_to_string(value) <> size_to_string(size)
         Call(Variable(size), [Field(None, value)]) ->
-          size_to_string(value) <> size
+          size_value_to_string(value) <> size_to_string(size)
         p -> string_to_string(p)
       }
     }
     _ -> string.inspect(param)
   }
 }
 
-fn size_to_string(value) {
+fn size_to_string(size: String) -> String {
+  case size {
+    "percent" -> "%"
```

Comment:
I haven't found a better way to do that, since the name of the size function is "percent" and the value for the size variable comes from the AST